### DISCUSSION TITLE
Continued confusion between "Issues" and "Discussions" from consumers of repo

### DISCUSSION THREAD

**Discussion by Makuna** [community/community#40363] [Projects and Issues]
### Select Topic Area

Product Feedback

### Body

It has continued to be a problem where consumers (not contributors) of my repos will post in "Issues" rather than use "Discussions" section.  While there is a feature to convert an issue into a discussion, its gets tedious to do this so often.

I believe the problem is that the banner for when creating an issue doesn't provide an overview of what an issue is compared to a discussion.  I have tried to make it clear in the text I can change, but it continues to be ignored as they only see it after making the first selection of type of issue.

**Comment by ossanna16** [comment]
Hi there @Makuna and welcome to our community! Thank you for the feedback!

**Comment by conorriches** [comment]
This is kind of fascinating because the word "issue" means totally different things to the two different groups of people.

To a customer:
* if they have a problem or an issue with the software, they report their issue by creating an issue. They don't want a discussion - they have equated issue with "problem" or a "support ticket".
* discussion probably sounds like an internal area where team talk is done. They'll likely stay out, if there are dragons they'll live under "discussion". They may also worry they'll be pestered with notifications if they start a discussion.

To the team:
* an issue is actually a unit of work that needs doing. User problems need to be discussed before being refined into issues.
* discussion is where things are worked out. If a customer raises an issue they may not be needed for any further discussion on the matter, as the discussion is for the team to solve the issue. 

So the current language is entirely from a developers point of view.

|  | Customer | Collaborator |
| ------------ | ------------ | ------------ |
| Issue | problem | unit of work |
| Discussion | internal talk, dragons | talking | 

Reframing langauage from a customer's point of view might work?
* Issue could become Task, customers might hesitate creating a task for the team as that doesn't feel like their role.
* Discussion could remain or be softened a bit to "Collaborate"?

I haven't thought much more about the suggested words but it's a fantastic example of language meaning different things to different people!

**Comment by Makuna** [comment]
@conorriches nails the core problem.  Add language differences to this makes it worse.
Further, due to history of Github where Discussions feature wasn't even present and all we had was Issues, many people were just taught it was the only place to go.

On top of better nomenclature, here are my suggestions and I will use one of my Arduino libraries Github as an example.
Today, when the user selects "Issues", they see a simple list of the templates.
![image](https://user-images.githubusercontent.com/7972357/205365610-1878c98f-78fd-46ca-92ec-a04c5c2e1bf9.png)
Note my solution of specific templates and a general one that once opens informs the user to STOP and go to discussions.  I find users still selecting Bug or Feature request and never selecting general template.  Further they will use the Blank option listed at the very bottom which I can't hide.  

Below I show an example of what I would like to see.  The top banner explaining what an issue is and suggesting a discussion.  And also remove the ability to create a blank issue.  Both Repo configurations?
![image](https://user-images.githubusercontent.com/7972357/205367272-e3c33518-acb1-49bf-8d7a-e149425fea1a.png)

**Comment by jfullstackdev** [comment]
`Issues` came first before `Discussions`

actually, I requested it to the Support Team through an email and they confirmed that they were working with `Discussions`

the reason why I requested it is because the `Issues` carries the wrong message when you just want to discuss something that is not an issue, that's the main reason 

as for your situation, `Discussions` is just optional to be enabled in a repo, meaning there is someone on the team that enabled it or you, maybe for another reason, 

if your consumers are accustomed to using `Issues` so be it or if you will be adding `Discussions`, there should be a really good information why you are adding it, so that they will not be confused

**Comment by Poopooracoocoo** [comment]
Also see https://github.com/dear-github/dear-github/issues/293
